src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleLegacy.apkdesc

```diff
@@ -4,75 +4,75 @@
     "AndroidManifest.xml": {
       "Size": 2584
     },
-    "res/drawable-mdpi-v4/icon.png": {
-      "Size": 2200
-    },
-    "res/drawable-hdpi-v4/icon.png": {
-      "Size": 4762
-    },
-    "res/drawable-xhdpi-v4/icon.png": {
-      "Size": 7462
-    },
-    "res/drawable-xxhdpi-v4/icon.png": {
-      "Size": 13092
-    },
-    "res/drawable-xxxhdpi-v4/icon.png": {
-      "Size": 20118
-    },
-    "res/layout/main.xml": {
-      "Size": 544
-    },
-    "resources.arsc": {
-      "Size": 1724
-    },
-    "classes.dex": {
-      "Size": 316700
-    },
-    "assemblies/UnnamedProject.dll": {
-      "Size": 2851
-    },
     "assemblies/Java.Interop.dll": {
-      "Size": 67719
+      "Size": 67758
     },
     "assemblies/Mono.Android.dll": {
-      "Size": 246165
+      "Size": 246233
     },
     "assemblies/mscorlib.dll": {
-      "Size": 769192
+      "Size": 769324
     },
     "assemblies/System.Core.dll": {
-      "Size": 28168
+      "Size": 28186
     },
     "assemblies/System.dll": {
-      "Size": 9163
+      "Size": 9171
     },
-    "lib/arm64-v8a/libxamarin-app.so": {
-      "Size": 19696
-    },
-    "lib/arm64-v8a/libmonodroid.so": {
-      "Size": 268792
+    "assemblies/UnnamedProject.dll": {
+      "Size": 2862
     },
-    "lib/arm64-v8a/libxa-internal-api.so": {
-      "Size": 65312
+    "classes.dex": {
+      "Size": 316760
     },
     "lib/arm64-v8a/libmono-btls-shared.so": {
       "Size": 1613872
     },
-    "lib/arm64-v8a/libmonosgen-2.0.so": {
-      "Size": 4050176
-    },
     "lib/arm64-v8a/libmono-native.so": {
       "Size": 707024
     },
-    "META-INF/ANDROIDD.SF": {
-      "Size": 2225
+    "lib/arm64-v8a/libmonodroid.so": {
+      "Size": 281352
+    },
+    "lib/arm64-v8a/libmonosgen-2.0.so": {
+      "Size": 4037584
```

Comment:
For that matter, how did `libmonosgen-2.0.so` *shrink* by ~12KB?

Reply:
`libmonosgen-2.0.so` is not built by this NDK. Runtime uses its own version of it (I think it's 19c?)

Reply:
A past dotnet/installer bump could have made `libmonosgen-2.0.so` smaller, but wasn't enough to fail the test.

Reply:
particularly since the test really only cares about the overall `.apk` size, not the sizes of entries within the `.apk`, iirc.

Reply:
It can fail if a single file changes enough: https://github.com/xamarin/xamarin-android/pull/6118#discussion_r682906288

```
Error: apkdiff: File 'assemblies/UnnamedProject.dll' has changed by 358 bytes (10.14 %). This exceeds the threshold of 5.00 %.
```